### DISCUSSION TITLE
Change README.md with locate

### DISCUSSION THREAD

**Discussion by ghost** [community/community#40570] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi all

#### Concept
![image](https://user-images.githubusercontent.com/111701513/205136884-29a35786-fc29-484d-9432-60a5ff189a67.png)

*As we can see this image we can change the language of README.md through the popup.*

**Comment by Mod8124** [comment, marked as answer]
Hi there!
First option, it's just a redirect to another readme for example `<a src="https://github.com/yourusername/yourusername/blob/main/README-Spanish.md" ></a>`
In this option you have to change the content manually

Second options, it's using github actions here you have good repository about [https://github.com/dephraiim/translate-readme](https://github.com/dephraiim/translate-readme)
